src/utils.js

```diff
@@ -287,6 +303,40 @@ genotet.utils.keySet = function(collection) {
 };
 
 /**
+ * Validates email address.
+ * @param {string} email
+ * @return {boolean}
+ */
+genotet.utils.validateEmail = function(email) {
```

Comment:
I think creating a utils function is necessary. Because it makes changing validation setting much easier.